fdbclient/StatusClient.actor.cpp

```diff
@@ -462,13 +462,15 @@ ACTOR Future<StatusObject> statusFetcherImpl( Reference<ClusterConnectionFile> f
 	// This could be read from the JSON but doing so safely is ugly so using a real var.
 	state bool quorum_reachable = false;
 	state int coordinatorsFaultTolerance = 0;
-	state Reference<AsyncVar<Optional<ClusterInterface>>> clusterInterface(new AsyncVar<Optional<ClusterInterface>>);
+
+	if(!clusterInterface) {
+		clusterInterface = Reference<AsyncVar<Optional<ClusterInterface>>>(new AsyncVar<Optional<ClusterInterface>>);
+		state Future<Void> leaderMon = monitorLeader<ClusterInterface>(f, clusterInterface);
```

Comment:
I think this works, but declaring the `state` variable in the `if` block may be a little confusing.